.gitmodules

```diff
@@ -43,3 +43,6 @@
 [submodule "third_party/xxHash"]
```

Comment:
That's exactly the reason why I had to delete `%Z` the other time - even the simplest `%z` which should simply show the UTC offset as +00:00 is not working reliably across BSD-related operating systems. And might produce `-00:00`(!!) for some weird reason, due to current DST settings and `tzname[]` content. 

I used to have simple `%FT%T` (not `%FT%T%z`), but then reimplemented our own `strftime()` which reliably using our own `tzoffset` instead of heuristics in LIBC, and restored `%z`. But then incorrectly copied to this patch, upon conflict resolution. Sorry for the confusion. Simplified again...

```diff
--- a/test/app-tap/datetime.test.lua
+++ b/test/app-tap/datetime.test.lua
@@ -175,7 +175,7 @@ test:test("Datetime string formatting", function(test)
     test:is(t.tzoffset, 0, ('t.tzoffset == %d'):format(t.tzoffset))
     test:is(t:format('%d/%m/%Y'), '01/01/1970', '%s: format #1')
     test:is(t:format('%A %d. %B %Y'), 'Thursday 01. January 1970', 'format #2')
-    test:is(t:format('%FT%T%z'), '1970-01-01T00:00:00+0000', 'format #3')
+    test:is(t:format('%FT%T'), '1970-01-01T00:00:00', 'format #3')
     test:is(t:format(), '1970-01-01T00:00:00Z', 'format #6')
     assert_raises(test, expected_str('datetime.strftime()', 1234),
                   function() t:format(1234) end)
```